PyInstaller/hooks/hook-weasyprint.py

```diff
@@ -0,0 +1,7 @@
+import os
```

Comment:
Please add the copyright header like in the other hook files.
Please add comments containing the URL of the package and about what version and what OS this hook is tested with